csm/conf/etc/csm/csm.conf

```diff
@@ -76,6 +76,7 @@ STATS:
         ssl_check: "false"
         interval: "10" # Flush interval in sec (Rate at which stats is stored)
         offset: "20" # offset in sec
+    telemetryAuth: "false"
```

Comment:
If you are not using this key - > 'telemetryAuth'.. then please remove it.

Reply:
Current code changes will not require it. Removed them.